assemblies/assembly-audit-log.adoc

```diff
@@ -3,29 +3,25 @@
 = Audit logs in {product}
 :context: assembly-audit-log
 
-Audit logs are a chronological set of records documenting the user activities, system events, and data changes that affect your {product} users, administrators, or components. Each audit log contains the following information:
+Audit logs are a chronological set of records documenting the user activities, system events, and data changes that affect your {product} users, administrators, or components. Administrators can view {product-short} audit logs in the {ocp-short} web console to monitor scaffolder events, changes to the RBAC system, and changes to the Catalog database. Audit logs include the following information:
 
-* Event name
-* Event actor. For example, terminal, port, location, IP address, or hostname.
-* Event metadata. For example, date, time, and other miscellaneous information.
-* Event status. For example, `success`, `failure`.
-* Severity levels. For example, `info`, `debug`,  `warn`, `error`.
+* Name of the audited event
+* Actor that triggered the audited event, for example, terminal, port, IP address, or hostname
+* Event metadata, for example, date, time
+* Event status, for example, `success`, `failure`
+* Severity levels, for example, `info`, `debug`,  `warn`, `error`
 
 You can use the information in the audit log to achieve the following goals:
 
 Enhance security::
-Trace activities, including those initiated by robots and software templates, back to their source. Know when software templates are executed, as well as the details of application and component installations, updates, configuration changes, and removals.
+Trace activities, including those initiated by automated systems and software templates, back to their source. Know when software templates are executed, as well as the details of application and component installations, updates, configuration changes, and removals.
```

Comment:
[SME] can we replace _robots_ with something like _automated systems_ or _automation systems_?

Reply:
I guess `automated systems` would make sense since in this context, it's just the backend processing a queue of tasks.